pkgs/development/python-modules/librosa/default.nix

```
@@ -9,20 +9,23 @@
 , audioread
 , resampy
 , soundfile
+, pooch
 }:
 
 buildPythonPackage rec {
   pname = "librosa";
-  version = "0.7.2";
+  version = "0.8.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "656bbda80e98e6330db1ead79cd084b13a762284834d7603fcf7cf7c0dc65f3c";
+    sha256 = "0fdqlbz3mrm0zc60zvvbrfbw6lv4q4kwv928qk5sxxmvshp9y2xg";
   };
 
-  propagatedBuildInputs = [ joblib matplotlib six scikitlearn decorator audioread resampy soundfile ];
+  propagatedBuildInputs = [ joblib matplotlib six scikitlearn decorator audioread resampy soundfile pooch ];
 
   # No tests
+  # 1. Internet connection is required
+  # 2. Got error "module 'librosa' has no attribute 'version'"
   doCheck = false;
```

This will catch the previous error
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "librosa" ];
```

---

Didn't know about that, thanks a lot!

---

This could be enabled by default for itself, wouldn't it ?

---

it could if all packages exported a modules of the same name, however, many do not.